charts/drax/templates/secret-influxdb2.yaml

```
@@ -0,0 +1,13 @@
+{{- $existingSecret := lookup "v1" "Secret" .Release.Namespace "drax-influxdb2-auth" -}}
+{{- if not $existingSecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: drax-influxdb2-auth
+  annotations:
+    "helm.sh/resource-policy": keep
+type: Opaque
+data:
+  admin-token: {{ randAlphaNum 48 | b64enc | quote}}
+  admin-password: {{ randAlphaNum 48 | b64enc | quote }}
```

We might want to change this to be a fallback (so if password and/or token is actually defined it would be used):

```suggestion
  admin-token: {{ $.Values.influxdb2.adminUser.token | default (randAlphaNum 48) | b64enc | quote}}
  admin-password: {{ $.Values.influxdb2.adminUser.password | default (randAlphaNum 48) | b64enc | quote }}
```

---

Ok